pr_agent/servers/bitbucket_server_webhook.py

```diff
@@ -13,35 +14,55 @@
 from pr_agent.agent.pr_agent import PRAgent
 from pr_agent.config_loader import get_settings
 from pr_agent.log import get_logger
+from pr_agent.servers.utils import verify_signature
 
 router = APIRouter()
 
 
-def handle_request(background_tasks: BackgroundTasks, url: str, body: str, log_context: dict):
+def handle_request(
```

Comment:
Why did you change the parenthesis styling?

Reply:
Black... I linted and formatted with black